src/react/hooks/__tests__/useSuspenseQuery.test.tsx

```diff
@@ -9363,14 +9344,12 @@ describe("useSuspenseQuery", () => {
           },
         },
         networkStatus: NetworkStatus.error,
-        error: new ApolloError({
-          graphQLErrors: [
-            new GraphQLError(
-              "homeWorld for character with ID 1000 could not be fetched.",
-              { path: ["hero", "heroFriends", 0, "homeWorld"] }
-            ),
-          ],
-        }),
+        error: new CombinedGraphQLErrors([
+          new GraphQLError(
+            "homeWorld for character with ID 1000 could not be fetched.",
+            { path: ["hero", "heroFriends", 0, "homeWorld"] }
+          ),
+        ]),
```

Comment:
```suggestion
        error: new CombinedGraphQLErrors([
          {
            message: "homeWorld for character with ID 1000 could not be fetched.",
            path: ["hero", "heroFriends", 0, "homeWorld"]
          },
        ]),
```

Similar for other places here - these are not `GraphQLError` instances and we should probably not test for that.